.gitignore

```diff
@@ -4,6 +4,8 @@
 /.nyc_output
 /node_modules
 /out
+/asm-docs*
+/static/dist
```

Comment:
I'm probably forgetting what's going on here, but I can't see how `/static/dist` is being created anywhere either?

Reply:
If I'm honest, I don't know how that one came about. It might be because I'm running it on a windows machine.  It didn't show up in the Linux build I have running on Ubuntu 20.  It might have come up during the node installation process.  I'll remove it, since after I deleted the directory, it didn't seem to come back. Here's a screenshot of what's inside it:
![image](https://user-images.githubusercontent.com/10420943/110183588-cfeebb00-7dd4-11eb-8cf2-31664318d610.png)

Reply:
`/static/dist` used to be a thing - but we moved it a long time ago.
It is likely your source tree just had left over stale files.

Reply:
Ah, that would make sense.  I had the repository from end of 2019 when I was working on TI's C2000 compiler.  I'll remove that from the ignore.